core/src/test/java/org/elasticsearch/search/SearchModuleTests.java

```diff
@@ -99,16 +99,16 @@ public void testRegisteredQueries() throws IOException {
         IndicesQueriesRegistry indicesQueriesRegistry = module.getQueryParserRegistry();
         XContentParser dummyParser = XContentHelper.createParser(new BytesArray("{}"));
         for (String queryName : supportedQueries) {
-            indicesQueriesRegistry.lookup(queryName, dummyParser, ParseFieldMatcher.EMPTY);
+            indicesQueriesRegistry.lookup(queryName, ParseFieldMatcher.EMPTY, dummyParser::getTokenLocation);
```

Comment:
And maybe it doesn't matter. If we're going to allocate an object we should just pass the position. But if we don't want to allocate we can just declare some interface and have the parser implement it and pass that.